benchmark/syclblas/blas3/gemm_batched_strided.cpp

```diff
@@ -186,17 +187,21 @@ void register_benchmark(blas_benchmark::Args& args,
     int t1 = static_cast<int>(blas_benchmark::utils::to_transpose_enum(t1s));
     int t2 = static_cast<int>(blas_benchmark::utils::to_transpose_enum(t2s));
 
+    index_t stride_a = m * k;
+    index_t stride_b = k * n;
+    index_t stride_c = m * n;
+
     auto BM_lambda = [&](benchmark::State& st, blas::SB_Handle* sb_handle_ptr,
```

Comment:
Looks like strides are not captured/passed properly inside the lambda function since it's not evaluated in-place within `RegisterBenchmark`. Passing the strides explicitly as arguments _(to be added in line 203 for as well)_ so that they're properly picked by `run<scalar>(..).` solved the observed segfaults and invalid strides.